### PR TITLE
Make Argo CD naming consistent

### PR DESCRIPTION
**Pull request by tedmiston**
This small PR changes instances of `ArgoCD` to `Argo CD` throughout the repo, and also fixes a few small typos.

Note: There are a few *.pb.go that are generated and I believe should not have been edited, however I'm not sure where I should edit upstream from the generation process.  Can you make a suggestion on that?

---

On the official Argo CD product page, docs, and throughout most of this repo, the name is spelled "Argo CD".

- https://argoproj.github.io/argo-cd
- https://blog.argoproj.io/introducing-argo-cd-declarative-continuous-delivery-for-kubernetes-da2a73a780cd

### CODE REVIEW

**Comment by tedmiston** [comment]
I've signed the CLA above, so I believe this PR is now ready for review.

File: server/settings/settings.proto [hunk 1]
```diff
@@ -3,13 +3,13 @@ option go_package = "github.com/argoproj/argo-cd/server/settings";
 
 // Settings Service
 //
-// Settings Service API retrives ArgoCD settings 
+// Settings Service API retrives Argo CD settings
```
Comment (alexmt) [inline server/settings/settings.proto]: Please apply the same change in https://github.com/argoproj/argo-cd/blob/master/server/settings/settings.pb.go#L9

**Review by alexmt — CHANGES_REQUESTED** [review]
Thank you for making this cleanup, @tedmiston !  The *.pb.go files are generated using `make codegen`  command (upstream files are *.pb and https://github.com/argoproj/argo-cd/blob/master/pkg/apis/application/v1alpha1/types.go) but it is fine to make changes manually. 

You've missed the only one autogenerated file: https://github.com/argoproj/argo-cd/blob/master/server/settings/settings.pb.go#L9

**Comment by tedmiston** [comment]
@alexmt Thank you for reviewing.  I resolved the typo and also freshly rebased off of upstream master.  Please let me know if any other revisions should be made.

**Comment by tedmiston** [comment]
The single word for SEO makes sense.

FWIW, it was me browsing your product site first and seeing `Argo CD` and not `ArgoCD` everywhere, then seeing mixed usage in the codebase that motivated this PR.

[I'd also be happy to take this PR in the other direction if `ArgoCD` is the current preference.]

Sample Google query results:

- `"argo cd" site:argoproj.io` ~= 15
- `"argocd" site:argoproj.io` = 0
- `"argo cd" site:github.*` ~= 1100
- `"argocd" site:github.*` ~= 773

**Comment by tedmiston** [comment]
Just wanted to touch base on this PR for canonicalizing the naming to see if any action is required on my end.

**Comment by jessesuen** [comment]
> Just wanted to touch base on this PR for canonicalizing the naming to see if any action is required on my end.

I've been overruled. Argo CD will be the correct usage going forward and not ArgoCD. Once the conflicts are resolved, it should be good to go.

**Review by alexmt — APPROVED** [review]

**Comment by tedmiston** [comment]
Thanks for checking @jessesuen and approving @alexmt.

I've rebased off master and resolved the conflicts.  I believe this PR is good to go.

**Pull request merged**